common/remote/sockfile.cpp

```diff
@@ -4537,7 +4537,7 @@ void verifyMetaInfo(IPropertyTree &actNode, bool authorizedOnly, const IProperty
     Owned<IPropertyTree> metaInfoEnvelope = createPTree(decompressedMetaInfoMb);
 
     Owned<IPropertyTree> metaInfo;
-#if defined(_USE_OPENSSL) && !defined(_WIN32)
```

Comment:
This change is not a warning fix

Reply:
Right, this change is actually in another ticket, so this will not be here where rebase to the version with that ticket.